v3data/hypes/impermanent_data.py

```diff
@@ -0,0 +1,441 @@
+
+import asyncio
+from datetime import timedelta
+
+from v3data import GammaClient, LlamaClient, UniswapV3Client
+from v3data.hypes.fees import Fees
+
+from v3data.utils import timestamp_ago
+from v3data.constants import BLOCK_TIME_SECONDS
+from v3data.hypes.fees_yield import FeesYield
+
+
+
+class ImpermanentData(FeesYield):
+
+    async def _get_hypervisor_data_at_block(self, block, hypervisors=None):
```

Comment:
Can we removes these from the class? since we are inheriting FeesYield which should already have these methods.

Reply:
query's differ .. no TVL involved in fees...
we could add an optional fields to   _get_hypervisor_data_at_block( query="", variables="") so we can reuse...

Reply:
let's just replace the query in Fees with this one then. I think they are sufficiently similar.

Reply:
TODO